src/pymedphys/cli/dicom.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2019 Cancer Care Associates
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version (the "AGPL-3.0+").
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License and the additional terms for more
+# details.
+
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+# ADDITIONAL TERMS are also included as allowed by Section 7 of the GNU
+# Affero General Public License. These additional terms are Sections 1, 5,
+# 6, 7, 8, and 9 from the Apache License, Version 2.0 (the "Apache-2.0")
+# where all references to the definition "License" are instead defined to
+# mean the AGPL-3.0+.
+
+# You should have received a copy of the Apache-2.0 along with this
+# program. If not, see <http://www.apache.org/licenses/LICENSE-2.0>.
+
+
+from ..dicom import (
+    adjust_machine_name_cli, adjust_RED_cli,
+    adjust_RED_by_structure_name_cli)
+
+
+def dicom_cli(subparsers):
```

Comment:
docs showing intended use?

Reply:
changelog?

Reply:
Cheers :) I'll put this back to WIP and I have been wanting to add in a suite of documentation + changelogs for the logfile side of things. Especially considering I plan to make the trf decode side of it public soon.

I'll land a heap of docs in this branch. Now's a better time than any other :)

Reply:
Yeah. I've actually been thinking that we ought to really spend some time updating and prettifying the docs.

I'm now starting the think docs are essential - almost as important as the code itself. Physicists will rely very heavily on the documentation when adopting PyMedPhys. This is something that James Kerns has done *very* well with `pylinac`.